stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/message/attachment/AttachmentUploader.kt

```diff
@@ -54,11 +45,12 @@ internal class AttachmentUploader(
                     it
                 }
             }
-
-            progressTracker?.setComplete(true)
+            attachment.uploadState = Attachment.UploadState.Success
+            progressCallback?.onSuccess(file.absolutePath)
```

Comment:
This would be a good time to document the methods of `ProgressCallback`, as it's not obvious what `String` you'll receive here as the parameter.

Actually, now that I'm commenting on this, it'd make more sense to me to get the URL of the successful upload than the local file path.

Reply:
I was not sure about the method of ProgressCallback if it accepts file path or URL. We should definitely document such things, especially it is a public class.

Reply:
It was done before Amit. By @zsmb13 ))) 
I mean this PR suppose to improve flow of upload progress. We can discuss and improve later this side of `ProgressCallback`. I believe no one currently has clear vision why it was done so